docs-gen/content/rule_set/overlay.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Overlay"
+date: 2019-08-04T12:46:30+02:00
+weight: 7
+---
+
+VSS defines the standard catalog for vehicle signals independent of the used protocol and environment.
+In order to adopt the specification we realize that certain additions and modifications to the standard catalog are necessary.
+**VSS Overlay** are meant to bring you a standardized way of handling those changes.
+
+The following features with the intended usage patterns are currently supported:
+1. **Adding new nodes:** By adding nodes the standard catalog can be extended with proprietary concepts.
+1. **Changing the value of existing metadata:** The standard catalog defines metadata based on what is assumed to be an average vehicle. 
+Configurations may differ slightly (e.g. the instantiation of number of available seats), or special situations that require a (limited) modification of existing metadata.
+1. **Adding new key/value pairs as additional metadata:** 
+Extending the model with richer information is a fundamental feature enabled by the layer concept.  For example, deploying VSS into a specific scenario or with a particular binding/technology often needs some additional information.
+1. **Multiple layer files:** VSS layers can be split into several files in order to clearly separate concerns. Layering allows all the features above to be applied in a composable manner. In order to keep a determinstic result a clear order has to remain.
+
+### Scenarios for using Overlays
+
+The standard catalogue is specified within COVESA and defines a common view of the major
+attributes, sensors and actuators of vehicles. This is used in many different scenarios,
+protocols and environments. Additional meta data might be required for data-governance,
+-quality or -sources. The instanciation of branches might not fit your vehicle.
+Proprietrary signals might be needed for extending the specification for your own use-cases.
+Up to now you were on your own in these scenarios. Overlays give you now the chance to change
+and modify the standard catalog in a standardized way.
+
+### How does it work?
+
+Simply said, the tooling takes next to the original specification file, *n* additional
```

Comment:
Simply said, the tooling accepts, *n* additional
 spec files, next to the original specification file,  which can overwrite or extend data in the VSS tree defined by the original specification.